### ISSUE TITLE
Timeout in setting FGS params

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/hyperion#886]
We had an issue where Hyperion hit the 10 minute timeout that GDA is expecting from it. The logs showed:
![image](https://github.com/DiamondLightSource/hyperion/assets/10086797/3931faa1-d9ad-43a3-a93f-cfc68e2c60dc)
From looking at the code this seems to be that we correctly set the FGS params (as confirmed by the epics log) but we did not get to waiting for them to be valid. This implies that we're stuck waiting for the put callback.

**Comment by DominicOram** [comment]
Split the parameter set into individual sets, waiting on logs to see which one might have caused the failure

**Comment by DominicOram** [comment]
Caused by https://github.com/DiamondLightSource/hyperion/issues/895 so closing this in favour of that.
